### DISCUSSION TITLE
Can't use seamless-scroll-polyfill with nextJs

### DISCUSSION THREAD

**Discussion by AnastasiusVivaldus** [vercel/next.js#29231] [Help]
### What version of Next.js are you using?

10.1.2

### What version of Node.js are you using?

14.17.0

### What browser are you using?

Safari

### What operating system are you using?

MacOS

### How are you deploying your application?

next dev

### Describe the Bug

Tried using https://github.com/magic-akari/seamless-scroll-polyfill by adding 

```
import { elementScrollIntoViewPolyfill } from "seamless-scroll-polyfill";
elementScrollIntoViewPolyfill();
```

... to `/src/_app.tsx`. This results in a "error - Error [ERR_REQUIRE_ESM]: Must use import to load ES Module ... " error.

It seems to me that this sort of "Must use import to load ES Module" issue affects a [bunch of libraries](https://github.com/vercel/next.js/issues/28567), and that, rather than nextjs fixing the root issue, we are expected to find a workaround for each library.

### Expected Behavior

Nextjs comes configured to be able to import from ESM libraries

### To Reproduce

Try installing seamless-scroll-polyfill into a nextjs typescript project.

**Comment by magic-akari** [comment]
https://github.com/magic-akari/seamless-scroll-polyfill/issues/131
Improved compatibility with next@8 ~ next@12
